chowda/views.py

```diff
@@ -128,14 +129,19 @@ class ClamsEventView(ModelView):
 
 
 class DashboardView(CustomView):
-    def sony_ci_last_sync(self):
-        # TODO: replace with actual "last sync" time
-        return datetime.now()
+    def sync_history(self) -> Dict[str, Any]:
+        return [
+            {'created_at': sync_run.created_at, 'successful': sync_run.successful}
+            for sync_run in list(Flow('IngestFlow'))
```

Comment:
When running in production, this errored with: `metaflow.exception.MetaflowNotFound` and crashed the server.

We should:
- [ ] Protect this block from not finding a flow
- [ ] Grab the correct results from `namespace('production:ingestflow-0-sdjk')`